dataactcore/models/stagingModels.py

```diff
@@ -528,6 +528,7 @@ class DetachedAwardFinancialAssistance(Base):
     business_funds_indicator = Column(Text)
     business_types = Column(Text)
     cfda_number = Column(Text)
+    cfda_title = Column(Text)
```

Comment:
It shouldn't be adding to the detached one, we aren't asking them to include the title in their file, we're deriving it on submit